### PR TITLE
Pt 154800940 python tests swagger 231

### PR DESCRIPTION
**Pull request by velzevur**

### CODE REVIEW

**Comment by ThomasArts** [comment]
Just a side node... it could be that swagger.json is not completely compatible with the swagger.yaml in the branch you merge it with later on!

**Comment by velzevur** [comment]
Rebased

File: config/swagger.yaml [hunk 1]
```diff
@@ -1461,7 +1461,7 @@ definitions:
     - data_schema
   SingleTxObject:
     allOf:
-      - $ref: '#/definitions/TxHashOrObject'
+      - $ref: '#/definitions/SingleTxHashOrObject'
```
Comment (lucafavatella) [inline config/swagger.yaml]: I see this main change.

Comment (velzevur) [reply config/swagger.yaml]: Yes, this and a couple of modifications in python code

**Comment by lucafavatella** [comment]
I see two approvals. I see CI failed.
I see no way this PR may conflict with PR #659 recently merged to master.

**Pull request merged**